sdk/typescript/src/cryptography/publickey.ts

```diff
@@ -108,7 +109,10 @@ export class PublicKey {
    * Return the Sui address associated with this public key
    */
   toSuiAddress(): string {
-    const hexHash = sha3_256(this.toBytes());
+    let tmp = new Uint8Array(PUBLIC_KEY_SIZE + 1);
+    tmp.set([TYPE_BYTE]);
```

Comment:
Will this always be `0x00` regardless of the flag?

Reply:
Thanks for the review Chris,
no, 00 is for ed25519, 01 will be for ECDSA k1 etc

Reply:
Sorry I missed the reply earlier. Does that mean we should change the function signature from `toSuiAddress()` to `toSuiAddress(schem: SignatureScheme = '0x00')`?